spec/plsql/spec/cli_spec.rb

```diff
@@ -91,6 +91,10 @@ def delete_test_files
       expect(File.directory?(@root_dir + '/spec/factories')).to be_truthy
     end
 
+    it "should create .rspec" do
```

Comment:
Added test to check that .rspec file is actually copied when calling `plsql-spec init`